src/content/blog/astro-210.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: "Astro 2.1 Release"
+description: "Introducing core image support • Markdoc integration • Watch mode for astro check • getStaticPaths() TypeScript helpers"
+publishDate: "March 8, 2023"
+authors:
+  - matthew
+coverImage: "/src/content/blog/_images/astro-210/blog-hero-2_1-release.webp"
+socialImage: "/src/content/blog/_images/astro-210/og-image-2_1-release.jpg"
+lang: "en"
+---
+
+import markdoc from "/src/content/blog/_images/astro-210/markdoc.webp"
+import astrocheckwatch from "/src/content/blog/_images/astro-210/astro-check-watch.webp"
+
+import BlogContentImage from "/src/components/BlogContentImage.astro"
+
+We just released Astro 2.1, featuring:
+
+- **[Image optimizations in core](#image-support-in-astro-core)**: An experimental feature that brings the learnings from `@astrojs/image` to core.
```

Comment:
Thoughts on "Built-in Image Support" vs. "Image Optimizations in core"? I think that reads a bit better.

"Built-in Image Optimization" or "Built-in Image Improvements" could also work.

Also, thoughts on adding `(experimental)` to the bolded title, so that it's more clear that this is experimental? 

```suggestion
- **[Image optimizations in core](#image-support-in-astro-core)**: An experimental feature that brings the learnings from `@astrojs/image` to core.
+ **[Built-in Image Support (experimental)](#built-in-image-support)**: The best parts of `@astrojs/image` are now in Astro core
```